sql/derived_tables/po_instance.sql

```diff
@@ -1,63 +1,80 @@
-DROP TABLE IF EXISTS po_instance;
+-- Create a local table for purchase order line instance. Every po line will have location ID or holding ID 

```

Comment:
This looks like the same derived table that was with your PR for instance_administrative_notes. Is that intentional?